services/brig/brig.cabal

```diff
@@ -16,8 +16,6 @@ copyright:      (c) 2017 Wire Swiss GmbH
 license:        AGPL-3
 license-file:   LICENSE
 build-type:     Custom
-data-files:
-    docs/swagger.md
```

Comment:
and why did you modify the cabal file instead of package.yaml?

Reply:
This only ended up by mistake in the cabal file, not in the package.yaml file.